src/Client/Main.hs

```diff
@@ -159,7 +159,7 @@ mainProgram cmd = case cmd of
     case pkgs of
       Left err -> printTextLn err
       Right packages -> runMultisigContract packages
-  CmdDeployContract (arg #owner -> mOwner) deployOptions -> do
+  CmdDeployContract (N mOwner :: "owner" :! Maybe AddressOrAlias) deployOptions -> do
```

Comment:
I'd argue that the type signature is unnecessary here, but I'm also not particularly against it.